guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +30,44 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Containers can be pushed to {Project}'s container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so content can be pushed to Katello using the same methods that apply to other registries.
+For more information about OCI, see https://opencontainers.org/[Open Container Initiative Distribution Specification].
```

Comment:
@asteflova I think the introduction of the module makes sense since, as you previously mentioned, the module's introduction covers only the previous scope of content.